.buildkite/pipeline.yml

```diff
@@ -1,45 +0,0 @@
-steps:
```

Comment:
`tests/RunTests.hs` now contains dangling references to `.circleci` and `.buildkite` in comments that need to be fixed.